### PR TITLE
style the button more like mbta.com's

### PR DESCRIPTION
**Pull request by thecristen**
I wasn't sure if you'd prefer PRs vs my pushing this in. This
- Adjusts the Tailwind "brand" color to Dotcom's "brand-primary"
- Sets the right font family (will have to figure out actually adding the Inter font later)
- Adjusts the button style to be more like Dotcom's
- Adjusts the button component to the same structure as the one in Phoenix's `CoreComponents`, so we can now configure the button type and add other classes and attributes.

### CODE REVIEW

File: lib/blocks/components/button.ex [hunk 1]
```diff
@@ -1,13 +1,31 @@
...
+
+    <.button>Send!</.button>
+    <.button phx-click="go" class="ml-2">Send!</.button>
+  """
 
   use Phoenix.Component
 
+  attr :type, :string, default: nil
+  attr :class, :string, default: nil
+  attr :rest, :global, include: ~w(disabled form name value)
```
Comment (anthonyshull) [inline lib/blocks/components/button.ex:15]: 👍 I was just adding this in to test it.

File: lib/blocks/components/button.ex [hunk 2]
```diff
@@ -1,13 +1,31 @@
...
+  ## Examples
+
+    <.button>Send!</.button>
+    <.button phx-click="go" class="ml-2">Send!</.button>
+  """
 
   use Phoenix.Component
 
+  attr :type, :string, default: nil
+  attr :class, :string, default: nil
```
Comment (anthonyshull) [inline lib/blocks/components/button.ex:14]: Let's remove this for now and try to think of a good way to allow for variants without giving people the ability/responsibility to change the design.

File: assets/tailwind.config.js [hunk 3]
```diff
@@ -14,8 +15,11 @@ module.exports = {
   ],
   theme: {
     extend: {
+      fontFamily: {
+        sans: ['Inter', 'Helvetica Neue', ...defaultTheme.fontFamily.sans],
+      },
```
Comment (anthonyshull) [inline assets/tailwind.config.js:20]: Check how we did it with MyCharlie: https://github.com/mbta/my_charlie/blob/main/assets/tailwind.config.js

Also, thinking we might do something like publish an npm package with just the colors just like:

```
const colors = require("tailwindcss/colors");
```

Comment (thecristen) [reply assets/tailwind.config.js:20]: I've got a half-baked version of that for the design system, I'll tackle color importing another time.

**Comment by thecristen** [comment]
Mostly obsolete, closing this

**Pull request closed without merging**